src/main/daml/Daml/Finance/Util/Date/DayCount.daml

```diff
@@ -3,7 +3,6 @@
 
 module Daml.Finance.Util.Date.DayCount
   ( calcDcf
-  , calcDcf30360
```

Comment:
Just pointing out that this is a breaking change to the API as we are making an existing function "disappear"

Reply:
> Just pointing out that this is a breaking change to the API as we are making an existing function "disappear"

If that is a problem, I can always keep the old function but have it call the new `calcDcf30Fix`. That way, we would get both code reuse and an intact API. What do you think?

Reply:
I think that makes sense!